src/py/flwr/cli/new/templates/main.pytorch.py.tpl

```diff
@@ -0,0 +1 @@
+"""A Flower / PyTorch app."""
```

Comment:
Why are there templates for a `main.py`, but `flower.toml` refers to `server.py` and `client.py`?

Reply:
This is yet mostly copy/paste. As mentioned in Slack we still need to work out the templates. The "flower.toml" as of now also has no real meaning as its not used anywhere.